src/components/LightUpChinatown/DonationTierItem.tsx

```diff
@@ -0,0 +1,195 @@
+import React from 'react';
+import styled from 'styled-components';
+import { useTranslation } from 'react-i18next';
+import {
+  tabletScreens,
+  phoneScreens,
+} from '../../utilities/general/responsive';
+
+interface Props {
+  imageUrl: string;
+  ctaText: string;
+  titleText: string;
+  descriptionItems: string[];
+  buttonText: string;
+  currentDonationCount: number;
+  maxDonationCount: number;
+}
+
+const DonationTierItem = (props: Props) => {
+  const { t } = useTranslation();
+  const remainingDonationCount =
+    props.maxDonationCount - props.currentDonationCount;
+
+  return (
+    <React.Fragment>
```

Comment:
If you're wrapping a component under a single node with styling like you do with Container, theres no need for React.fragment.